Tests/SwiftGRPCNIOTests/NIOServerTests.swift

```diff
@@ -300,3 +300,89 @@ extension NIOServerTests {
     waitForExpectations(timeout: defaultTimeout)
   }
 }
+
+// Only test Unary and ServerStreaming, as ClientStreaming is not
+// supported in HTTP1.
+extension NIOServerTests {
```

Comment:
Would it make sense to put these tests into a separate file with a separate test case object, WDYT?

Reply:
Done